src/backend/src/controllers/projects.controllers.ts

```diff
@@ -279,15 +267,7 @@ export const editProject = async (req: Request, res: Response) => {
       slideDeckLink,
       bomLink,
       taskListLink,
-      rules,
-      wbsElement: {
-        update: {
-          name,
```

Comment:
name, project lead, and prjoect manager still need to be updated. we're only taking out status here